src/config.ts

```diff
@@ -92,6 +98,12 @@ const defaultConfig: Config = {
   media: { enabled: false, requireAuth: true, requirePubkeyInRule: false },
   list: { requireAuth: false, allowListOthers: false },
   tor: { enabled: false, proxy: "" },
+  whitelist: {
+    enabled: true,
```

Comment:
```suggestion
    enabled: false,
```
Should default to false